styles-gen/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "styles-gen",
```

Comment:
should we just call this `immich-map-static` something like that instead?

Reply:
This is only for the styles, so not sure. The fonts and sprites aren't handled yet but probably wouldn't be handled in this repository, as they can just be pulled statically from protomaps repo.

Reply:
`immich-styles` seems better than `styles-gen` to me at least, but w/e you want.

Reply:
Sure I can change it to that